tests/tasks/test_light.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import unittest
+
+import parlai.utils.testing as testing_utils
+from parlai.agents.repeat_label.repeat_label import RepeatLabelAgent
+from parlai.core.worlds import create_task
+from parlai.scripts.display_data import setup_args
+
+NUM_EPS = 17075
+NUM_EXS = 110877
```

Comment:
Are these the values of the dataset when split on a score cutoff? I'm not sure where these come from or when we may need to change them.